Contants/Texts/Source/texts/Skills_BattleOrder.txt

```diff
@@ -87,4 +87,12 @@ P-Flow[X]
 ## MSG_SKILL_PassionsFlow
 Passions Flow:[NL]
 Unit doubles if currently[NL]
-gaining support bonuses.[X]
\ No newline at end of file
+gaining support bonuses.[X]
+
+## MSG_SKILL_CloseCombat_NAME
+C-Combat[X]
+
+## MSG_SKILL_CloseCombat
+Close Combat:[NL]
++2 damage if at 1 range. Your[NL]
+double attacks occur before countering.[X]
```

Comment:
is the skill description too long to display on statscreen rtext or prep skill acteen

Reply:
![image](https://github.com/user-attachments/assets/f50b45d6-c11e-46ad-8394-61647b532f99)

Reply:
Fair enough, will fix 

![image](https://github.com/user-attachments/assets/af484ed3-bd65-4ef2-9021-b4edc71f547e)